### PR TITLE
[17.03] Bump 17.03.2

### PR DESCRIPTION
**Pull request by cpuguy83**

### CODE REVIEW

**Comment by cpuguy83** [comment]
ping @mlaventure @thaJeztah

**Comment by runcom** [comment]
👍

**Comment by mlaventure** [comment]
No `rc`? :)

Also, `CHANGELOG.md`  :angel:

**Comment by mlaventure** [comment]
- Updated version to `17.03.2-ce-rc1`
- Added initial `CHANGELOG`

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,28 @@ information on the list of deprecated flags and APIs please have a look at
 https://docs.docker.com/engine/deprecated/ where target removal dates can also
 be found.
 
+## 17.03.1-ce (2017-05-24)
```
Comment (andrewhsu) [inline CHANGELOG.md]: version `17.03.2-ce` and future date to `2017-05-29` to give time for rc cycle

**Review by andrewhsu — CHANGES_REQUESTED** [review]
future date changelog version

**Review by andrewhsu — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,28 @@ information on the list of deprecated flags and APIs please have a look at
...
+## 17.03.2-ce (2017-05-29)
+
+### Networking
+
+- Fix a concurrency issue preventing network creation [#33273](https://github.com/moby/moby/pull/33273)
+
+### Runtime
+
+- Relabel secrets path to avoid a Permission Denied on selinux enabled systems [#33236](https://github.com/moby/moby/pull/33236)
++ Add a new mount `z` option to instruct that a local volume need to be relabeled [#33236](https://github.com/moby/moby/pull/33236)
```
Comment (cpuguy83) [inline CHANGELOG.md]: I haven't looked at the pr yet as I'm on mobile, but this should just be fixing some edge axes around the "z"option. The option itself has been around for awhile.

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,28 @@ information on the list of deprecated flags and APIs please have a look at
...
+- Fix an issue while upgrading if a plugin rootfs was still mounted [#33236](https://github.com/moby/moby/pull/33236)
+- Fix an issue where volume wouldn't default to the `rprivate` propagation mode [#33236](https://github.com/moby/moby/pull/33236)
+- Fix a panic that could occur when a volume driver could not be retrieved [#33236](https://github.com/moby/moby/pull/33236)
++ Add a warning in `docker info` when the `overlay` or `overlay2` graphdriver is used on a filesystem without `d_type` support [#33236](https://github.com/moby/moby/pull/33236)
+- Prevent volume to be deleted if its spec cannot be backported [#33207](https://github.com/moby/moby/pull/33207)
```
Comment (cpuguy83) [inline CHANGELOG.md]: "Fix an issue with backporting mount spec to older volumes"

**Comment by mlaventure** [comment]
@cpuguy83 updated

**Review by cpuguy83 — COMMENTED** [review]
LGTM

**Comment by mlaventure** [comment]
ping @thaJeztah

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,28 @@ information on the list of deprecated flags and APIs please have a look at
...
+- Fix an issue where volume wouldn't default to the `rprivate` propagation mode [#33236](https://github.com/moby/moby/pull/33236)
+- Fix a panic that could occur when a volume driver could not be retrieved [#33236](https://github.com/moby/moby/pull/33236)
++ Add a warning in `docker info` when the `overlay` or `overlay2` graphdriver is used on a filesystem without `d_type` support [#33236](https://github.com/moby/moby/pull/33236)
+- Fix an issue with backporting mount spec to older volumes [#33207](https://github.com/moby/moby/pull/33207)
+- Fix issue where a failed unmount can lead to data loss on local volume remove [#33120](https://github.com/moby/moby/pull/33120)
+
+### Swarm Mode
+
+- Fix a case where tasks could get killed unexpectedly [#33118](https://github.com/moby/moby/pull/33118)
+- Fix an issue preventing to deploy services if the registry cannot be reached despite the needed images being locally present [#33117://github.com/moby/moby/pull/33117)
```
Comment (tiborvass) [inline CHANGELOG.md]: typo `s/:/]/`

File: CHANGELOG.md [hunk 5]
```diff
@@ -5,6 +5,28 @@ information on the list of deprecated flags and APIs please have a look at
...
 
+## 17.03.2-ce (2017-05-29)
+
+### Networking
+
+- Fix a concurrency issue preventing network creation [#33273](https://github.com/moby/moby/pull/33273)
+
+### Runtime
+
+- Relabel secrets path to avoid a Permission Denied on selinux enabled systems [#33236](https://github.com/moby/moby/pull/33236)
```
Comment (tiborvass) [inline CHANGELOG.md]: I understand the link to the cherrypick PR, but it's weird. I would put the link to the original PR instead. Or both?

Comment (tiborvass) [inline CHANGELOG.md]: If the cherrypick PR didn't cherrypick all in one, I would prefer the cherrypick PR since it's closest to what actually is changed in the release branch. But in this case it bundles too much and it's hard to follow.

Comment (mlaventure) [inline CHANGELOG.md]: I had initially put the original one, but it leads to the wrong branch. I'll put both with `ref` I guess

**Comment by mlaventure** [comment]
Fix the typo, and added the reference PR as needed if the fix was cherry-picked along side other fixes

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!
